README.md

```diff
@@ -1,6 +1,97 @@
 # Medical Event Data Standard
 
-The Medical Event Data Standard (MEDS) is a draft data schema for storing streams of medical events, often sourced from either Electronic Health Records or claims records.
+The Medical Event Data Standard (MEDS) is a data schema for storing streams of medical events, often
+sourced from either Electronic Health Records or claims records. Before we define the various schema that make
+up MEDS, we will define some key terminology that we use in this standard.
+
+## Terminology
+  1. A _patient_ in a MEDS dataset is the primary entity being described by the sequences of care observations
+     in the underlying dataset. In most cases, _patients_ will, naturally, be individuals, and the sequences
+     of care observations will cover all known observations about those individuals in a source health
+     datasets. However, in some cases, data may be organized so that we cannot describe all the data for an
+     individual reliably in a dataset, but instead can only describe subsequences of an individual's data,
+     such as in datasets that only link an individual's data observations together if they are within the same
+     hospital admission, regardless of how many admissions that individual has in the dataset (such as the
+     [eICU](https://eicu-crd.mit.edu/) dataset). In these cases, a _patient_ in the MEDS dataset may refer to
+     a hospital admission rather than an individual.
+  2. A _measurement_ or _patient measurement_ or _observation_ in a MEDS dataset refers to a single measurable
+     quantity observed about the patient during their care. These observations can take on many forms, such as
+     observing a diagnostic code being applied to the patient, observing a patient's admission or transfer
+     from one unit to another, observing a laboratory test result, but always correspond to a single
+     measureable unit about a single patient.
+  3. A _code_ is the categorical descriptor of what happened in a patient measurement. In particular, in
+     almost all structured, longitudinal datasets, a measurement can be described as consisting of a tuple
+     containing a `patient_id` (who this measurement is about); a `timestamp` (when this measurement
```

Comment:
Are we doing time or timestamp? I think the current schema uses "time", which I would recommend because it's unambigious yet also shorter.

Reply:
I have no preference; I default to timestamp as "time" can refer to just the "time" part of a "datetime" but it is not a strong preference, so I'll use "time" unless I hear further from you.